model_compression_toolkit/core/common/mixed_precision/sensitivity_evaluation.py

```diff
@@ -283,7 +284,8 @@ def _compute_distance_matrix(self,
                                                   framework_attrs=self.interest_points[i].framework_attr,
                                                   compute_distance_fn=self.quant_config.compute_distance_fn)
 
-            distance_matrix[i] = point_distance_fn(baseline_tensors[i], mp_tensors[i], batch=True)
+            axis = self.interest_points[i].framework_attr.get(AXIS, None)
```

Comment:
get default is None. No nned to specify it

Reply:
I know I just like it when it's more explicit this way, cause you know what to expect when the key doesn't exist, but I'll change it